src/test/java/org/simulator/sbml/SBMLTestSuiteTest.java

```diff
@@ -159,7 +165,73 @@ public void testModel() throws FileNotFoundException, IOException, XMLStreamExce
                 duration = timepoints[timepoints.length - 1]
                         - timepoints[0];
 
-                if (model.getExtension(CompConstants.shortLabel) == null){
+                if (model.getExtension(CompConstants.shortLabel) != null) {
+
+                    // initialize simulator
+                    CompSimulator compSimulator = null;
+                    boolean errorInCompSimulator = false;
+                    try {
+                        compSimulator = new CompSimulator(sbmlFile);
+                    } catch (Exception e) {
+                        errorInCompSimulator = true;
```

Comment:
This is not good. Why we would suppress the exception in the test? That is why we write the tests. If everything is working - we are satisfied, if not - we want to know the reason. Please feel free to throw an exception in the tests. JUnit will treat this as an error and you will be able to see the stack trace to solve the problem fast and not spend your time guessing why this problem occurs.